backend/test/parser/TheParser.test.ts

```diff
@@ -270,6 +270,46 @@ test('parse expand tag', () => {
   )
 })
 
+test('parse gallery tag', () => {
+  // Basic gallery
+  expect(p.parse('<gallery><img src="https://orbitar.media/image1.jpg" /></gallery>').text).toEqual(
+    '<div class="gallery" show-arrows=true show-indicators="true" show-thumbnails="true" auto-play-interval="0"><img src="https://b.orbitar.media/image1.jpg" alt=""/></div>',
```

Comment:
> `show-arrows=true`

by convention, in html the presence of the attribute usually indicates `true`. meaning, either consider setting the defaults for the keys to be `false`, or inverting the names (e.g. `no-indicators`, `no-thumbnails`).

even better design, imo, is to remove all these options for the sake of simplicity and consistency of the UI (picking the best defaults or inferring them)